playbooks/roles/machine/tasks/install_machine_agent_linux.yml

```diff
@@ -147,3 +150,40 @@
       path: "{{ machine_agent_dest_folder_linux }}/{{ ma_agent_dest_file }}"
       state: absent
     changed_when: false
+
+
+  #Custom Extension Prep
+  - name: Create subdirectory for custom extension
+    file:
+      path: "{{ machine_agent_dest_folder_linux }}/monitors/customextensions"
```

Comment:
Creation of directory should be controller with a bool variable (similar to enable_analytics_agent parameter)
And deployment of custom extensions needs to be a separate task:

include_tasks: configure_custom_extensions_linux.yml
when: enable_custom_extensions | bool
Also, I suggest removing this from the current branch and creating a new feature branch with custom extensions script changes;